demo/web/src/main/notebooks/A. Real-time table ops/A6 Time methods.md

```diff
@@ -0,0 +1,83 @@
+# Time methods
+
+In our previous notebook, we showed how to use conditionals on time series data. In this notebook, we show how to use Deephaven's time methods on time series data.
+
+Let's start again by simulating some real-time data, but this time with a shorter time interval.
+
+```python
+from deephaven.DateTimeUtils import currentTime, expressionToNanos, minus
+
+time_interval = expressionToNanos("T1S")
+offset = expressionToNanos("T1000S")
+now = currentTime()
+
+daily_data = create_random_table(time_interval, start_time=minus(now, offset))
+```
+
+## where
+
+`where` filters can be applied to time series data. This example shows a few ways to filter your time series data.
+
+```python
+even_seconds = daily_data.where("secondOfMinute(Timestamp, TZ_NY) % 2 == 0")
+odd_seconds = daily_data.where("secondOfMinute(Timestamp, TZ_NY) % 2 == 1")
+
+even_minutes = daily_data.where("minuteOfHour(Timestamp, TZ_NY) % 2 == 0")
+odd_minutes = daily_data.where("minuteOfHour(Timestamp, TZ_NY) % 2 == 1")
+```
+
+## update
```

Comment:
I don't really consider where or update time methods. The minuteOfHour etc qualify. Downsampling should probably be in here, too. I'd say some calendar stuff, too, but we never documented that.

Reply:
I agree, I added a downsampling example